en/concept/flight_modes.md

```diff
@@ -14,23 +14,23 @@ The sections below provide an overview of the modes, followed by a [flight mode
 "Manual" modes are those where the user has direct control over the vehicle via the RC control (or joystick). Vehicle movement always follows stick movement, but the level/type of response changes depending on the mode. For example, experienced fliers can use modes that provide direct passthrough of stick positions to actuators, while beginners will often choose modes that are less responsive to sudden stick-position changes.
 
 * **Fixed wing aircraft/ rovers / boats:** 
-  * **MANUAL:** The pilot's control inputs are passed directly to the output mixer.
-  * **STABILIZED:** The pilot's inputs are passed as roll and pitch *angle* commands and a manual yaw command.
+  * **MANUAL:** The pilot's control inputs or RC commands are passed directly to the output mixer.
+  * **STABILIZED:** The pilot's inputs are passed as roll and pitch *angle* commands and a manual yaw command. If the RC roll and pitch sticks are centered, the autopilot regulates the roll and pitch angles to zero, hence stabilizing (leveling-out) the attitude against any wind disturbances. However, in this mode the position of the aircraft is not controlled by the autopilot, hence the position can drift due to wind.
```

Comment:
@bkueng Is this correct?

@barzanisar Assuming this is correct, it is an excellent update.  Essentially you're saying that this keeps the vehicle upright if roll/pitch sticks are centred - that wasn't clear before. 

What does the first sentence actually imply/mean? ("The pilot's inputs are passed as roll and pitch *angle* commands and a manual yaw command.") 
- ie how does this improve stabilisation?

Reply:
In the first line, I don't understand what "**manual** yaw command means" and how it is different from the "roll and pitch commands". Can anyone please explain this? Does this mean that the "roll and pitch commands" from RC are sent as references to the autopilot's attitude controller which tries to track these references, whereas the the "yaw command" from RC is sent directly as output to the rudder to control yaw manually? 

If this is true, the first line can be further elaborated as:
"The pilot's inputs are passed as roll and pitch *angle* commands to the autopilot and a yaw command to control rudder manually."

Reply:
Lets see what @bkueng says :-)

Reply:
Ok, so I had to ask for that as well :)

> Does this mean that the "roll and pitch commands" from RC are sent as references to the autopilot's attitude controller which tries to track these references

Yes this is correct.

> the "yaw command" from RC is sent directly as output to the rudder to control yaw manually?

That is correct too. But there is one thing missing: the vehicle does a [coordinated turn](https://en.wikipedia.org/wiki/Coordinated_flight), to have zero sideslip (the acceleration in y-direction (sidewards) is zero). This means during a turn, the rudder is used to control the sideslip. If the pilot gives non-zero yaw input, the input will just be added to what the vehicle commands for the coordinated turn.

Reply:
@barzanisar Doc on coordinated turn should be added.

Reply:
@bkueng do you mean that in stabilised mode the aircraft always does a coordinated turn for nonzero roll/pitch/yaw inputs?

Reply:
> do you mean that in stabilised mode the aircraft always does a coordinated turn for nonzero roll/pitch/yaw inputs?

Yes, that's how I understand it.

> @bkueng in https://docs.px4.io/en/flight_modes/ I found that for fixed wing aircrafts, the state of ACRO mode decides whether to switch to Stabilised mode or Manual mode. And from the Dev guide's flight mode evaluation diagram I understood the following, so correct me if I am wrong:

This is incorrect (outdated) and needs to be updated. There is now a separate acro mode for fixed-wing: the user controls the vehicle's body rates and the controller stabilizes the rates (all 3 axis: roll, pitch & yaw).
(whereas stabilize will level out and do the coordinated turns, so stabilize the attitude angles)